src/platform.h.in

```diff
@@ -64,10 +65,22 @@ namespace ncnn {
 class NCNN_EXPORT Mutex
 {
 public:
-    Mutex() { InitializeSRWLock(&srwlock); }
```

Comment:
it may blame the restyled formatter
keep the rest of this file unchanged.

Reply:
checked out master 👍

Reply:
but we need the new cmakedefine01 NCNN_AVX    :smile: